### ISSUE TITLE
Browser (web interface) doesn't work due to missing assets

### ISSUE THREAD

**Issue by obahareth** [RailsEventStore/rails_event_store#583]
I have a new Rails app and I followed the instructions in [the tutorial](https://railseventstore.org/docs/browser/), and I get these errors:

```
ActionController::RoutingError (No route matches [GET] "/res/ruby_event_store_browser.css"):
```

```
ActionController::RoutingError (No route matches [GET] "/res/ruby_event_store_browser.js"):
```

**Comment by pawelpacana** [comment]
Do you perhaps use git dependency in `Gemfile`? Assets are compiled and packaged in released gems.

**Comment by obahareth** [comment]
Oh yes, you're absolutely right. I had to do that to try out the fix for #578. Is that fix released yet?

**Comment by pawelpacana** [comment]
Fix hasn't been released yet but I guess that will eventually happen this week
